PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -45,9 +45,30 @@ struct SettingsForm: View {
     var body: some View {
         Form {
             Group {
-                topSectionWithLeadingDivider()
```

Comment:
It's not clear to me why are we removing this now?

Reply:
good catch. i missed it during the rebase. ill fix that in the next commit